individual/schema.py

```diff
@@ -0,0 +1,46 @@
+import graphene
+from django.contrib.auth.models import AnonymousUser
+from django.db.models import Q
+
+from core.schema import OrderedDjangoFilterConnectionField
+from core.utils import append_validity_filter
+from individual.apps import IndividualConfig
+from individual.gql_mutations import CreateIndividualMutation, UpdateIndividualMutation, DeleteIndividualMutation
+from individual.gql_queries import IndividualGQLType
+from individual.models import Individual
+import graphene_django_optimizer as gql_optimizer
+
+
+class Query:
+    individual = OrderedDjangoFilterConnectionField(
+        IndividualGQLType,
+        orderBy=graphene.List(of_type=graphene.String),
+        dateValidFrom__Gte=graphene.DateTime(),
+        dateValidTo__Lte=graphene.DateTime(),
+        applyDefaultValidityFilter=graphene.Boolean(),
+        client_mutation_id=graphene.String()
+    )
+
+    def resolve_individual(self, info, **kwargs):
+        filters = []
+        filters += append_validity_filter(**kwargs)
```

Comment:
I think we can make it shorter
```suggestion
        filters = append_validity_filter(**kwargs)
```